debezium-connector-mongodb/src/test/java/io/debezium/connector/mongodb/FieldRenamesIT.java

```diff
@@ -1485,10 +1489,17 @@ private SourceRecord getUpdateRecord(String database, String collection, String
     }
 
     private void assertDocumentContainsFieldError(String fieldName) {
-        stopConnector(value -> {
-            final String message = "IllegalArgumentException: Document already contains field : " + fieldName;
-            assertThat(logInterceptor.containsStacktraceElement(message)).isTrue();
-        });
+        final String message = "IllegalArgumentException: Document already contains field : " + fieldName;
+        try {
+            Awaitility.await().atMost(Duration.ofSeconds(30))
```

Comment:
@Naros Thanks, I wonder if it makes sense to introduce `io.debezium.connector.postgresql.TestHelper.waitTimeForRecords()` for MongoDB as well and make it multiplication of the standard wait time instead of hardcoded constant.

Reply:
That makes sense, I'll add that and resync the PR.